content/pages/sprint/2022/12-hsinchu.rst

```diff
@@ -0,0 +1,76 @@
+========================================
+Scisprint 2022 December in Hsinchu
+========================================
+
+:date: 2022-11-24 10:00
+:url: sprint/2022/12-hsinchu
+:save_as: sprint/2022/12-hsinchu.html
+
+Scisprint 2022 December in Hsinchu
+========================================
+
+* Date: 3rd December, Saturday, 2022
+* Time: 13:00 -- 17:00 (4 hours)
+
+To join the sprint, please bring your laptop and sign up in the `sprint note
+<#sprint-note>`__.  You are also very welcome to bring your project, which
+should be added in the `sprint note <#sprint-note>`__ too.  Please `contact us
+<#contact-us>`__ if you have any questions.
+
+Scisprint is an event providing space for people to work on open source
+projects together. We welcome any subjects that may interest the attendees,
+and especially encourage code for science, engineering, and technology, which
+demand more critical discussions than other applications of computer
+programming.
+
+In this sprint, we provide a `Sprint Training Room <#Sprint-Training-Room>`__ for the newcomers/less experienced developers to get prepared to contribute in the open-source projects.
+
+We would like to provide a supportive and friendly environment for all attendees to support more developers
+to join in the open-source communities. 
+
+
+Sprint Training Room
+--------------------
+
+If you uaually felt frustrated with starting to contribute in an open-source project or have no idea how to deal with it, 
+now it is a great time to join `Sprint Training Room <https://hackmd.io/XRFfX0zeQGqFedKfi6eqTg#Sprint-Training-room>`__.
+You will learn some git/GitHub skills which is commonly used in contributing workflow, which is an important 
+entry point to start your open-source path.
+
+For more information, you could check out in `Sprint Training Room <https://hackmd.io/XRFfX0zeQGqFedKfi6eqTg#Sprint-Training-room>`__.
+If you have any questions, feel free to join `Discord <https://discord.gg/6MAkFrD>`__ and interact with us.
+
+Cost sharing
+------------
+
+This occurrence is free.
+
+Venue
+-----
+
+`國立清華大學第三綜合大樓 A區 5樓 <https://goo.gl/maps/EH2wWtkLQ8qLWd669>`__. Please see the `sprint
+note <#sprint-note>`__ for the access of the venue.
+
+.. raw:: html
+
+  <div style="overflow:hidden; padding-bottom:56.25%; position:relative; height:0;">
```

Comment:
Could you make it as wide as the page?  Currently it's narrow:

<img width="1110" alt="image" src="https://user-images.githubusercontent.com/399122/203882268-3893a6f3-8942-4c76-9b3e-38b6d985ad74.png">

Perhaps you can take a look at https://sciwork.dev/sprint/2022/07-taipei .  The code is at https://github.com/sciwork/swportal/blob/f68ca3dcc17931c97459273f3ba994f898853334/content/pages/sprint/2022/07-taipei.rst?plain=1#L41

Reply:
Sure. I could fix it.